pkg/controller/control.go

```diff
@@ -211,6 +212,7 @@ func (c *Controller) Snapshot(name string, labels map[string]string) (string, er
 	if err := c.handleErrorNoLock(c.backend.Snapshot(name, true, created, labels)); err != nil {
 		return "", err
 	}
+	log.Infof("Finished snapshot")
```

Comment:
You can just use `Info` when you don't pass a format string arg.
Same for the other places.